src/docs/sdk/performance/index.mdx

```diff
@@ -32,6 +34,41 @@ Optionally, the `tracesSampler` callback can also return a boolean to force a sa
 
 See more about how sampling should be performed below.
 
+### `tracePropagationTargets`
+
+Sentry SDKs propagate trace information via headers of outgoing Http requests to downstream SDKs (see [`sentry-trace`](#header-sentry-trace) and [`baggage`](/sdk/performance/dynamic-sampling-context/#baggage) for more details).
+The `tracePropagationTargets` option gives users a mechanism of controlling to which outgoing Http requests these headers should be attached.
+For example, users can specify this property to keep trace propagation within their infrastructure, thereby stopping it from being sent to third party services.
+
+This option takes an array of strings or regular expressions to match against the URLs of outgoing requests.
+If this option is specified by users, SDKs may only add the trace headers to a request if the the request's URL matches (contains) at least one of the items from the array.
+
+#### Example
+
+```JS
+// Entries can be strings or RegEx
+tracePropagationTargets: ['localhost', /^\// ,/myApi.com\/v[2-4]/]
+
+URLs matching: 'localhost:8443/api/users', '/api/envelopes', 'myApi.com/v2/projects'
+URLs not matching: 'someHost.com/data', 'myApi.com/v1/projects'
+```
+
+#### Default Behaviour
+
+If the `tracePropagationTargets` option is not specified by users, SDKs should fall back to their default behaviour for trace propagation.
+This default behaviour is not standardized and can vary across SDKs.
+For instance, Server-side SDKs may propagate trace headers to all outgoing requests by default while client-side SDKs (e.g. Javascript) can fall back to a default array of allowed domains (e.g. `localhost` and `/^\//` in JavaScript).
+Choosing a sensible default behaviour is up to the SDK maintainers, however, special limitations, such as CORS issues for frontend web SDKs must be considered.
+SDKs that previously supported the `tracingOrigins` property (see deprecation notice below) can continue using the already established default behaviour for `tracePropagationTargets`
```

Comment:
Given my thoughts about defaults, I think we'd be fine doing away with this section entirely and just adding a sentence or two above the example, something like

> SDKs may choose a default value which makes sense for their use case. Most SDKs default to the regex `/.*/` (attach headers to all outgoing requests), but deviation is allowed if necessary. (For example, because of CORS, browser-based SDKs default to only adding headers to internal requests.)

WDYT?

(If you do go this route, feel free to use or not use as much of the above as you'd like.)

Reply:
I think that's a good change. Took your example and adjusted it slightly